buildkite/src/Constants/ContainerImages.dhall

```diff
@@ -13,5 +13,5 @@
   nodeToolchain = "node:14.13.1-stretch-slim",
   ubuntu2004 = "ubuntu:20.04",
   xrefcheck = "serokell/xrefcheck@sha256:8fbb35a909abc353364f1bd3148614a1160ef3c111c0c4ae84e58fdf16019eeb",
-  nixos = "nixos/nix"
+  nixos = "gcr.io/o1labs-192920/nix-unstable:1.0.0"
```

Comment:
Jobs like Mina Artifact Migration * require ContainerImages.minaCaqtiToolchain. [REF ](https://github.com/MinaProtocol/mina/blob/4a3cc7c7cedb4035d6cedd13fd7d52eade572830/buildkite/src/Constants/ContainerImages.dhall#L11-L14)

Reply:
@SanabriaRusso @emberian i fixed above issue (and one more ). Rerunning nightly